travis/resource_cron.go

```diff
@@ -0,0 +1,195 @@
+package travis
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/shuheiktgw/go-travis"
+)
+
+func resourceCron() *schema.Resource {
+	return &schema.Resource{
+		Description: "The `travis_cron_job` resource creates a cron job for a branch.",
+
+		CreateContext: resourceCronCreate,
+		ReadContext:   resourceCronRead,
+		DeleteContext: resourceCronDelete,
+
+		Schema: map[string]*schema.Schema{
+			"repository_id": &schema.Schema{
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Description:  "Value uniquely identifying the repository.",
+				ForceNew:     true,
+				ExactlyOneOf: []string{"repository_slug"},
+			},
+			"repository_slug": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				Description:  "Same as {repository.owner.name}/{repository.name}.",
+				ForceNew:     true,
+				ExactlyOneOf: []string{"repository_id"},
+			},
+			"branch": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The branch to which this cron job belongs.",
+				ForceNew:    true,
+			},
+			"interval": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
```

Comment:
The serevr returns an error without interval.

> 422 error Invalid value for interval. Interval must be "daily", "weekly" or "monthly"!

```suggestion
				Required:     true,
```

Reply:
Nice catch.